inspect_wandb/config/settings.py

```diff
@@ -67,6 +67,7 @@ class WeaveSettings(BaseSettings):
     entity: str = Field(alias="WANDB_ENTITY", description="Entity to write to for the Weave integration")
 
     autopatch: bool = Field(default=False, description="Whether to automatically patch Inspect with Weave calls for tracing")
+    sample_display_name_template: str = Field(default="{task_name}-sample-{sample_id}-epoch-{epoch}", alias="SAMPLE_NAME_TEMPLATE", description="Template for sample display names. Available variables: {task_name}, {sample_id}, {epoch}")
```

Comment:
I don't think we want an alias here - the default env var of inspect_wandb_weave_sample_name_template would be more consistent and preferable, even if it's a bit lengthy

The alias should only be used in cases where we want to take advantage of env vars that might already be set in the users environment